### ISSUE TITLE
6th List of problems

### ISSUE THREAD

**Issue by supercoolsonic** [PiKeyAr/sadx_dreamcast#68]
1. If Sonic has a shield at the end of Windy Valley Act 3, sometimes, when he runs down the automated section, he won't be inside his shield.

2. Near one of the ruins you aligned, there's a collision problem.
![pitfall](https://cloud.githubusercontent.com/assets/26051785/23597634/1d80b6a8-01fa-11e7-8c56-7801346cdb6c.png)
Right about here, to the left of Sonic. This wasn't caused by moving the ruins, because I had this happen a few times when I first discovered the misalignment. It's minor, but it makes you fall through the wall.

3. In the Ice Cap icy room in Mystic Ruins, the "underwater effect" model in the pool...doesn't seem like it's fully there.
![broken effect](https://cloud.githubusercontent.com/assets/26051785/23597633/1d809178-01fa-11e7-96b3-e62d3dc1dbb3.png)

4. Sometimes, when using the elevator on the Egg Carrier to enter the inside area, the hatch will be closed, and Sonic will get stuck on in, but you'll still go inside.

**Comment by PiKeyAr** [comment]
I'm not very interested in fixing any of the above.

1 is likely a transparency sorting issue, which can't be fixed without a proper transparency solution.
2 is a problem with how SADX handles collision, it may or may not be fixable by putting some SET objects with collision there. In general, unless it's caused by my mods and breaks gameplay in a significant way, I'm not interested in fixing code or collision issues with the game.
3 happens on the Dreamcast (at least on the emulator) too, it's just the way the object is modeled. When I came across this issue some time ago, I tried to align it better, but it didn't work.
4 happens on the Dreamcast as well, it's probably a minor code issue with the original game.
